generators/Exercises/Hamming.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class Hamming : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in CanonicalData.Cases)
+            {
+                canonicalDataCase.ExceptionThrown = canonicalDataCase.Expected is long ? null : typeof(ArgumentException);
+            }
```

Comment:
I was a little curious why you chose long over int. Looking at the canonical data, I definitely would've gone int. But after debugging your branch a bit I noticed that it ultimately stems from JSON.NET parsing ints as int64 (http://json.codeplex.com/workitem/20832) -- so we do have to use long here.